syntax = "proto3";

package gps_data;

message GPS {
    int32	lat_deg		= 1;	// Целое число градусов широты
    int32	lat_flt		= 2;	// Дробное число градусов широты
    int32	lon_deg		= 3;	// Целое число градусов долготы
    int32	lon_flt		= 4;	// Дробное число градусов долготы
    int32	speed		= 5;	// Скорость устройства
    int32	direction	= 6;	// Направление устройства
    int32	precision	= 7;	// Точность определения координат
    int32	sat_qty		= 8;	// Число спутников GPS
    bool	is_move		= 9;	// Признак движения устройства
    uint32	ts		= 10;	// Время в секундах UTC, начиная с 01.01.2012
}
